Dashboard/Connectors/Serial/SerialConnector.cs

```diff
@@ -10,6 +11,10 @@ public class SerialConnector(ISerialPort comPort) : IConnector
     public event EventHandler<GpsData>? GpsDataUpdated;
     public event EventHandler<AvData>? AvDataUpdated;
     public event EventHandler<ResData>? ResDataUpdated;
+    public event EventHandler<RawData>? RawDataUpdated;
+    public event EventHandler<bool>? HeartBeatUpdated;
+    private Timer? _heartbeatTimer;
+    private const string HeartBeatMessage = "HeartBeat";
```

Comment:
According to the spec, this should just be `CON?`

Reply:
And the logger should return `OK` to signal that a device is connected to the logger

Reply:
are we meant to be heart beat checking PC -> AV logger, or AV logger -> car?

Reply:
> are we meant to be heart beat checking PC -> AV logger, or AV logger -> car?

we should be checking pc -> logger (serial connection)